### PR TITLE
fix: invalid yaml in openapi spec

### PR DESCRIPTION
**Pull request by ross-weir**

### CODE REVIEW

File: src/main/resources/api/openapi.yaml [hunk 1]
```diff
@@ -1718,10 +1716,12 @@ components:
             - $ref: '#/components/schemas/Timestamp'
         headersScore:
           type: integer
+          format: int64
```
Comment (kushti) [inline src/main/resources/api/openapi.yaml:1719]: could be out of 64 bits, so better to stay on format being skipped I guess

File: src/main/resources/api/openapi.yaml [hunk 2]
```diff
@@ -1718,10 +1716,12 @@ components:
...
           description: Can be 'null' if no headers is applied since node launch
           nullable: true
         fullBlocksScore:
           type: integer
+          format: int64
```
Comment (kushti) [inline src/main/resources/api/openapi.yaml:1724]: could be out of 64 as well

File: src/main/resources/api/openapi.yaml [hunk 3]
```diff
@@ -1701,7 +1699,7 @@ components:
           example: 327
         difficulty:
           type: integer
-          format: int32
+          format: int64
```
Comment (kushti) [inline src/main/resources/api/openapi.yaml:1702]: better to skip format, can be out of 64 bits

**Review by kushti — CHANGES_REQUESTED** [review]

File: src/main/resources/api/openapi.yaml (hunk 1 above)
Comment (ross-weir) [reply src/main/resources/api/openapi.yaml]: Removed & added a note to description

**Review by kushti — APPROVED** [review]

**Pull request merged**